action-server/covidflow/utils/maps.py

```diff
@@ -29,6 +29,24 @@ def get_map_url(coordinates: Coordinates, size: str = "220x150", zoom: int = 15)
     return f"{MAPS_ENDPOINT}{url_path}&signature={signature}"
 
 
+def get_static_map_url_from_address(
+    address: str, size: str = "220x150", zoom: int = 15
+):
+    key = os.environ[GOOGLE_API_KEY_ENV]
+    secret = os.environ[GOOGLE_SIGN_SECRET_ENV]
+    parameters = {
+        "markers": address,
```

Comment:
The only difference with this VS `get_static_map_url` is the value for this `markers` property - you can extract the rest in a single function